src/components/Modal/Example.md

```diff
@@ -287,3 +283,51 @@ class ModalConsumer extends React.Component {
 
 <ModalConsumer />
 ```
+
+**onBeforeClose**
```

Comment:
Can you give a meaningful name?

Reply:
like what? 
 - "onBeforeClose event?"
 - "controlling form close"
- [your option here]